generators/app/index.js

```diff
@@ -18,6 +18,11 @@ module.exports = Generator.extend({
     return this.prompt(prompts).then(function (options) {
       // To access options later use this.options.someAnswer;
       this.options = options;
+
+      // Compose
+      this.composeWith('one-base:gulp', {
```

Comment:
From what I saw yesterday, the syntax of this changed to be 
```
this.composeWith(require.resolve('../gulp'), {
  options: this.options
});
```

[Source](http://yeoman.io/blog/hello-generator-1.0.html)